pkg/apis/kubedirector/v1beta1/kubedirectorapp_types.go

```diff
@@ -27,12 +28,23 @@ type KubeDirectorAppSpec struct {
 	SchemaVersion       int                 `json:"configSchemaVersion"`
 	DefaultImageRepoTag *string             `json:"defaultImageRepoTag,omitempty"`
 	DefaultSetupPackage SetupPackage        `json:"defaultConfigPackage,omitempty"`
-	Services            []Service           `json:"services,omitempty"`
+	Services            []Service           `json:"services"`
 	NodeRoles           []NodeRole          `json:"roles"`
 	Config              NodeGroupConfig     `json:"config"`
-	DefaultPersistDirs  *[]string           `json:"defaultPersistDirs,omitempty"`
-	Capabilities        []corev1.Capability `json:"capabilities,omitempty"`
-	SystemdRequired     bool                `json:"systemdRequired,omitempty"`
+	DefaultPersistDirs  *[]string           `json:"defaultPersistDirs"`
+	Capabilities        []v1.Capability     `json:"capabilities"`
+	SystemdRequired     bool                `json:"systemdRequired"`
```

Comment:
Why the removal of all the omitempty tags?

Reply:
Yea got removed while resolving merge conflicts I guess. Will add back